.gitignore

```diff
@@ -3,3 +3,5 @@ node_modules/
 .vscode
 *.log
 sample/*/package-lock.json
+coverage/
+.nyc_output/
```

Comment:
@AlexLakatos Can you share why this is needed?

Reply:
Because I was working on test coverage in a different PR, and I forgot to not commit .gitignore. 😨